testutils/infra/device.py

```diff
@@ -57,7 +57,12 @@ def __init__(self, host_string="localhost:8822", user="root"):
             user=self.user,
             port=self.port,
             connect_timeout=60,
-            connect_kwargs={"password": "", "banner_timeout": 60, "auth_timeout": 60},
+            connect_kwargs={
+                "password": "",
+                "banner_timeout": 60,
+                "auth_timeout": 60,
+                "look_for_keys": False,
```

Comment:
So simple... I spend like an hour looking for this (months ago) and couldn't find such a setting...